libs/server/ACL/UserHandle.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Threading;
+
+namespace Garnet.server.ACL
+{
+    /// <summary>
+    /// A reference used to access a <see cref="User"/>.
+    /// </summary>
+    public class UserHandle
```

Comment:
why do we need a class (`UserHandle`) inside another class. seems a bit wasteful to keep around two objects per user.